### ISSUE TITLE
Unable to edit an existing form

### ISSUE THREAD

**Issue by chrisrowe** [verbb/formie#1180]
### Describe the bug

Form listing has no links http://up.chri.sr/Screen-Shot-2022-11-17-12-32-06.09-1668688328.png

### Steps to reproduce

Create a form, try to edit it

### Form settings

Used the built in stencil

### Craft CMS version

4.3.2.1

### Plugin version

2.0.17

### Multi-site?

No

### Additional context

Freeform installed but disabled

**Comment by andypullen** [comment]
I'm seeing this issue as well. It seems to be new to 4.3.2.1. Entries in the Submissions and Sent Notifications tables also do not have links.

**Comment by denisyilmaz** [comment]
same problem here after updating to Craft CMS 4.3.2.1

**Comment by chrisrowe** [comment]
worth noting the edit url actually works, `admin/formie/forms/edit/<FORMID>` the link just isn't in the listing

**Comment by umkasanki** [comment]
We have same issue

**Comment by sword3d** [comment]
I have the same problem

**Comment by engram-design** [comment]
Yeah, looks like Craft 4.3.x has broken things. Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by uandco** [comment]
Same here, works with Craft 4.3.1 but breaks from 4.3.2

**Comment by engram-design** [comment]
@uandco does the above fix not work for you?

**Comment by uandco** [comment]
> @uandco does the above fix not work for you?

it does, but I was being more precise about the Craft version breaking it, as comments suggested it was either 4.3.2.1 or 4.3.x, in case some want to downgrade instead of using the dev branch of Formie.

**Comment by engram-design** [comment]
Ah, gotcha! Yes, it was this commit https://github.com/craftcms/cms/commit/d7ad2a246356c79e749e86f4b1d7baafaa42f1fb in 4.3.2

**Comment by engram-design** [comment]
Fixed in [2.0.18](https://github.com/verbb/formie/releases/tag/2.0.18)
